hack/.deployUtils

```diff
@@ -321,13 +321,28 @@ deployPrometheusForFederation() {
 }
 
 installAPIDashboards() {
+  local PROMETHEUS_FOR_FEDERATION_API_DASHBOARDS_KUSTOMIZATION_DIR="$2"
+  local PROMETHEUS_FOR_FEDERATION_API_DASHBOARDS_GRAFANA_PATCH="$3"
+  if [ -z "$2" ]; then
+    PROMETHEUS_FOR_FEDERATION_API_DASHBOARDS_KUSTOMIZATION_DIR=${LOCAL_SETUP_DIR}/../config/prometheus-for-federation/api-dashboards
+  fi
+  if [ -z "$3" ]; then
```

Comment:
In a use case where you dont want to patch grafana this will still be populated I'm thinking could we add a extra check

Reply:
The patching is to mount the new dashboards.
I don't see a reason not to always do both.

Reply:
no your right i forgot how were were adding the dashboard using patch for the POC i was thinking adding new dashboards to existing one ignore me 😂